testcases/extensions-map.http

```diff
@@ -0,0 +1,10 @@
+POST / HTTP/1.1
+Host: handler.example.com
+Content-Length: 0
+ce-specversion: 0.4-wip
+ce-id: 123
+ce-source: https://cloudevents.io/extensions-map
+ce-type: io.cloudevents.extensions-map-test
+ce-this0is0ok-": ok
```

Comment:
I believe that `"` isn't allowed as an http header name. So this might fall into the category of what is mentioned in #478 

Of course, if we remove maps then this entire test goes away

Reply:
Yep, removed the `extensions-map` files.